src/election_anomaly/__init__.py

```diff
@@ -123,6 +123,7 @@ def track_results(self, shortname, election):
 
         if error != []:
             print(error)
+            print("metadata record not created in database")
             exit()
```

Comment:
I don't think we want to exit here, correct? The function should just return without doing any work?